docs/reference/overrides/dialog-trigger.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "DialogTrigger",
-  "attributes": {
-    "[data-modal]": {
+  "dataAttributes": {
+    "data-modal": {
```

Comment:
I was kind of deliberate about naming it "attributes" since in the old docs, I saw certain components exposed some `aria-*` attributes in the documentation, so it's not just data-attributes. Should we keep it this way?

Reply:
We agreed to only use custom data attributes for style hooks. I don't have a preference about `attributes` vs `dataAttributes` here, though.

Reply:
Cool I am probably mixing it up with non-style attributes that we had documented on some components
<img width="698" alt="image" src="https://github.com/user-attachments/assets/a0e64e46-c864-4b0d-802e-d1e212c48e4a">

Not sure how I feel about those, but that's separate

Reply:
I don't think we document any aria attributes we add, and yep, the rest of them are just props that we accept. I would keep `dataAttributes` to be explicit, if in the future we want to document other things, we anymay may want to re-think the API section.